okhttp-logging-interceptor/src/test/java/okhttp3/logging/HttpLoggingInterceptorTest.java

```diff
@@ -70,8 +70,8 @@
       new HttpLoggingInterceptor(applicationLogs);
 
   private void setLevel(Level level) {
-    networkInterceptor.setLevel(level);
-    applicationInterceptor.setLevel(level);
+    networkInterceptor.level(level);
+    applicationInterceptor.level(level);
```

Comment:
If it's not good to use `level(...)` this time, let's remove my commit. Or please rebase it into your commit it if the changes are fine :)

Reply:
We shouldn't be updating the java tests as part of these PRs, we lose an important check of public API checks.

Unfortunately these methods are excluded from the normal controls for other reasons

```
    // Became 'final' in 4.0.0:
    'okhttp3.logging.HttpLoggingInterceptor#redactHeader(java.lang.String)',
    'okhttp3.logging.HttpLoggingInterceptor#getLevel()',
    'okhttp3.logging.HttpLoggingInterceptor#setLevel(okhttp3.logging.HttpLoggingInterceptor$Level)'
```

Reply:
Sure! Deleted the test change commit :)